packages/next/src/build/index.ts

```diff
@@ -1735,9 +1735,17 @@ export default async function build(
       }
 
       if (Object.keys(functionsConfigManifest).length > 0) {
+        const manifest: {
+          version: number
+          functions: Record<string, Record<string, string | number>>
+        } = {
+          version: 1,
+          functions: functionsConfigManifest,
+        }
+
         await fs.writeFile(
           path.join(distDir, SERVER_DIRECTORY, FUNCTIONS_CONFIG_MANIFEST),
-          JSON.stringify(functionsConfigManifest, null, 2)
```

Comment:
Was this shipped in stable Next.js or was it only in canary?

Reply:
Was shipped into stable

Reply:
This will need to be considered in `@vercel/next` builder then because we don't want to error on a stable version of Next.js even if it was missing the version.

Reply:
Yes, I'll consider this in the next builder 👍